plugins/BEdita/Core/src/Model/Table/ObjectsTable.php

```diff
@@ -88,6 +87,7 @@ public function initialize(array $config): void
 
         $this->addBehavior('BEdita/Core.ObjectModel');
         $this->addBehavior('BEdita/Core.Categories');
+        $this->addBehavior('BEdita/Core.Status');
```

Comment:
I don't think we need the behavior here if it's already in `ObjectModelBehavior`